src/main/java/org/basepom/mojo/duplicatefinder/classpath/ClasspathDescriptor.java

```diff
@@ -73,6 +73,12 @@
         ".*overview\\.html$"
         ));
 
+    private static final MatchPatternPredicate DEFAULT_IGNORED_CLASS_PREDICATE = new MatchPatternPredicate(Arrays.asList(
+        // this regex matches nothing at all, just need something as a place holder
```

Comment:
It doesn't support empty arrays, but I've changed my code (included in next push) to use your inner class predicate as the default, which avoids the need for this.